etcd-manager/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.19-buster
```

Comment:
Why is this file required?

Reply:
It's only for dev purposes, it won't go in the final PR. I need it to be able to build an image containing both etcd-manager executable and sh since the command launching the etcd-manager image is generated by kops (hard to modify) and is in the form of `/bin/sh -c exec /etcd-manager [arguments]`

Reply:
Maybe it could be contributed in a different PR to etcdmanager? Maybe other people would benefit from it.